validation/src/main/scala/hmda/validation/dsl/PredicateRegEx.scala

```diff
@@ -26,6 +26,26 @@ object PredicateRegEx {
     override def failure: String = s"is not a valid phone number"
   }
 
+  implicit def validZipCode: Predicate[String] = new Predicate[String] {
+
+    override def validate: (String) => Boolean = {
+      val zipCodeRegex = "^\\d{5}(?:-\\d{4})?$".r
```

Comment:
Created an issue: https://github.com/cfpb/hmda-platform/issues/359.

I agree this shouldn't block this PR from being merged.